workflow-templates/npm-audit-fix.yml

```diff
@@ -47,8 +47,8 @@ jobs:
         run: npm i -g npm@"${{ steps.versions.outputs.npmVersion }}"
 
       - name: Fix npm audit
-        run: |
-          npm audit fix
+        id: npm-audit
+        uses: susnux/npm-audit-action@2a60bd2e79cc77f2cc4d9a3fe40f1a69896f3a87 # v0.1.0
```

Comment:
if this is used in nextcloud repos I would prefer moving https://github.com/susnux/npm-audit-action to the commong organization too. Else we run into the problem again where we find an issue with an action but have no permission to send a fix.

Reply:
Sure just created under my handle just like with skjnldsv permissions action.
So under which organization would you prefer it?

Reply:
fair point. we should move them all, also skjnldsv actions :wink: 

we can use https://github.com/nextcloud or https://github.com/nextcloud-libraries/ if we want more control over who can push

Reply:
cc @skjnldsv

Reply:
> also skjnldsv actions

except I am publishing them on the github store and they are not related to the NC workload :)

Reply:
So what's the conclusion? Should I move it to nextcloud or nextcloud-libraries? :)

Reply:
@ChristophWurst I agree, moved it.